extensions/amp-analytics/0.1/cookie-writer.js

```diff
@@ -87,24 +109,31 @@ export class CookieWriter {
     }
 
 
-    if (!hasOwn(this.config_, 'writeCookies')) {
+    if (!hasOwn(this.config_, 'cookies')) {
       return Promise.resolve();
     }
 
-    if (!isObject(this.config_['writeCookies'])) {
-      user().error(TAG, 'writeCookies config must be an object');
+    if (!isObject(this.config_['cookies'])) {
+      user().error(TAG, 'cookies config must be an object');
       return Promise.resolve();
     }
 
     this.registerDynamicBinding_();
-    const inputConfig = this.config_['writeCookies'];
+    const inputConfig = this.config_['cookies'];
+
+    if (inputConfig['enabled'] === false) {
+      // Enabled by default
+      // TODO: Allow indiviual cookie object to override the value
```

Comment:
>it might change the logic quite a bit. 

I don't agree. Especially if we are only going to support enable right now. It's only moving the check from here to within the for loop.